### PR TITLE
SPORTS-240 Add username and logout on TopBar

### PR DESCRIPTION
**Pull request by sanjeevkpandit**
image

### CODE REVIEW

File: admin-app/src/store/modules/user.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+// Actions
+const actions = {
+  fetchUserInfo(context: any) {
+    authService
+      .getUserInfo()
+      .then(response => {
+        context.commit('setUserInfo', response);
+      })
+      .catch();
```
Comment (SAGARACH65) [inline admin-app/src/store/modules/user.ts]: let's remove the unused catch block

File: admin-app/src/store/modules/user.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  },
+
+  logout(context: any) {
+    authService.logOut();
+  }
+};
+
+// Getters
+const getters = {};
```
Comment (SAGARACH65) [inline admin-app/src/store/modules/user.ts]: an empty cont objects?

File: api/src/mutations/user.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * @param {UserInfoPayload} payload
+ * @param {Context} context
+ * @returns {Promise<object>}
+ */
+export async function userInfo(parent: any, payload: UserInfoPayload, context: Context): Promise<object> {
+  if (context.error) {
+    throw new ApolloError(context.error, HttpStatus.FORBIDDEN.toString());
+  }
+
+  const info = await userService.getUserInfo(payload.refreshToken);
```
Comment (SAGARACH65) [inline api/src/mutations/user.ts]:
```suggestion
  return userService.getUserInfo(payload.refreshToken);
```

File: admin-app/src/store/modules/user.ts (hunk 2 above)
Comment (sanjeevkpandit) [reply admin-app/src/store/modules/user.ts]: Making it consistent everywhere as for the stores.

**Pull request merged**